SocketIOClientSwift/SocketTypes.swift

```diff
@@ -27,8 +27,9 @@ import Foundation
 public typealias AckCallback = ([AnyObject]) -> Void
 public typealias NormalCallback = ([AnyObject], SocketAckEmitter?) -> Void
 public typealias OnAckCallback = (timeoutAfter: UInt64, callback: AckCallback) -> Void
+public typealias SocketOptionsSet = Set<SocketIOClientOption>
```

Comment:
Opposed to what? I don't see a case where you'd have multiple items for the same option.